test/php.ini

```
@@ -1929,6 +1929,8 @@ opcache.restrict_api=/disabled/
 ; Absolute path used to store shared lockfiles (for *nix only).
 ;opcache.lockfile_path=/tmp
 
+opcache.jit=disable
```

https://php.watch/versions/8.0/JIT#jit-opcache.jit

off would be a bit better
But, can we debug the stituation and not ignore it ?
Because it would mean that real users can walk into this issue ?
~~Since this php.ini config is a sync of the Debian one~~
Edit: I just searched into git and it looks like my changes never did get into git. Well, maybe it's worth checking that are the default configs on other systems

---

Yeah, I opened it to see if the error go away. Looks like the type errors didn't happen. As @kamil-tekiela said, looks like a JIT bug.

---

Okay, let's try to not hide bugs in our testing tools. If users can have this bug using a default config, it's best to not hide or ignore it